api/src/main/java/io/opentelemetry/api/trace/DefaultTracerProvider.java

```diff
@@ -23,6 +24,7 @@ public Tracer get(String instrumentationName) {
 
   @Override
   public Tracer get(String instrumentationName, String instrumentationVersion) {
+    Objects.requireNonNull(instrumentationName);
```

Comment:
we can't add these requirements. The API must not throw exceptions in the face of bad inputs.